docs/modules/ROOT/pages/capacity-planning.adoc

```diff
@@ -585,8 +585,8 @@ of your Hazelcast data pipeline.
 
 === Upgrade Your JDK
 
-Hazelcast does not support JDK 8, so if you are still on JDK 8, please upgrade it.
-The garbage collectors of JDK 11+ have been improved to achieve much higher throughputs. 
+Hazelcast does not support JDK 8 and 11, so if you are still on these, please upgrade it.
```

Comment:
```suggestion
Hazelcast does not support JDK 8 or 11. If you are using either of these version, upgrade to version 17 or later.
```

Reply:
```suggestion
Newer JDK releases often contain performance improvements, consider upgrading to 
xref:deploy:versioning-compatibility.adoc#supported-java-virtual-machines[a newer supported version].
```

nit - shouldn't it just be <=11? We don't list 7, 9, 10 etc.
Does it make sense to list the supported JDKs on this page anyway? We have a dedicated page for that.
_(not sure the syntax of my suggestion is right)_

Reply:
I don't think it is necessary to list them here since, as you said, we have a dedicated page (linked from here)?